### DISCUSSION TITLE
Table 'app.notifications' doesn't exist

### DISCUSSION THREAD

**Discussion by krutyosila** [filamentphp/filament#11792] [Help]
### Package

Panel builder

### Package Version

v3

### How can we help you?

`SQLSTATE[42S02]: Base table or view not found: 1146 Table 'panel.notifications' doesn't exist.`

I tried all commands for notifications but I got this message. How can I migrate notifications table?

**Comment by leandrocfe** [comment]
run `php artisan notifications:table` and `php artisan migrate`

Check this section to enable the notifications in the panel
https://filamentphp.com/docs/3.x/panels/notifications
